lib/browser/chrome-extension.js

```diff
@@ -72,21 +72,28 @@ const backgroundPages = {}
 const startBackgroundPages = function (manifest) {
   if (backgroundPages[manifest.extensionId] || !manifest.background) return
 
-  const scripts = manifest.background.scripts.map((name) => {
-    return `<script src="${name}"></script>`
-  }).join('')
-  const html = new Buffer(`<html><body>${scripts}</body></html>`)
+  let html
+  let name = '_generated_background_page.html'
```

Comment:
Might be cleaner to set this in the `else` block after line `80` and just initialize to undefined like the `html` variable for consistency.